pkgs/applications/misc/monitor/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, vala
+, pkgconfig
+, pantheon
+, python3
+, gettext
+, glib
+, gtk3
+, bamf
+, libwnck3
+, libgee
+, libgtop
+, libxml2
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "monitor";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "stsdc";
+    repo = "monitor";
+    rev = version;
+    sha256 = "17z1m193s7qygavfwd8qsw97blxbfmq9gnsymdjlc1ddk8hldw0z";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    meson
+    ninja
+    vala
+    pkgconfig
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    bamf
+    glib
+    gtk3
+    pantheon.granite
+    pantheon.wingpanel
+    libgee
+    libgtop
+    libwnck3
+    libxml2
```

Comment:
```suggestion
```

Where is this needed? It builds without it.

Reply:
Meson doesn't have it in its dependencies but the README lists it as a dependency, which confused me. I'll remove it.